spec/core/string/include_spec.rb

```diff
@@ -13,6 +13,22 @@
     StringSpecs::MyString.new("hello").include?(StringSpecs::MyString.new("lo")).should == true
   end
 
+  it "returns true if both strings are empty" do
+    "".should.include?("")
+    # NATFIXME: Implement EUC-JP encoding
+    # "".force_encoding("EUC-JP").should.include?("")
+    # "".should.include?("".force_encoding("EUC-JP"))
+    # "".force_encoding("EUC-JP").should.include?("".force_encoding("EUC-JP"))
```

Comment:
We can still cover the intention of this spec by replacing EUC-JP with a different non-default encoding (e.g. one of the UTF16/UTF32 variants).